libdnf/utils/filesystem.hpp

```diff
@@ -30,6 +30,9 @@ namespace libdnf {
 bool
 pathExists(const char *path);
 
+bool
+pathExistsSafe(const std::string & path);
```

Comment:
Please could you add a description here. If you have only devel package available, then the description is hidden. Probably using word Safe is not the perfect one. Please could you find a better name?

Reply:
I'll add the description. Regarding the name, I've already thought about it and this is the best I have come up with, suggestions welcome.

Reply:
pathExists2